bears/python/MypyBear.py

```diff
@@ -0,0 +1,157 @@
+from collections import namedtuple
+import textwrap
+import sys
+import re
+import os
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.results.Result import Result
+
+OUTPUT_REGEX = (r'(?P<filename>[^:]+):((?P<line>\d+):)? '
+                '(?P<severity>[^:]+): (?P<message>.*)')
+
+
+class FlagInfo(namedtuple('FlagInfo', 'arg doc inverse')):
+    """Information about a command-line flag
+
+    :param arg:
+        The argument to pass to enable the flag
+    :param doc:
+        Help for the flag
+    :param inverse:
+        Set to true when the coala option is the inverse of
+        the subprocess one, for example coala's "allow_untyped_calls"
+        is the inverse of mypy's "--disallow-untyped-calls".
+    """
+
+    def want_flag(self, value):
+        """Check if the flag should be added to the argument list"""
+
+        if self.inverse:
+            value = not value
+        return value
+
+
+FLAG_MAP = {
+    'allow_untyped_functions': FlagInfo(
+        arg='--disallow-untyped-defs',
+        doc='Allow defining functions without type annotations or with '
+            'incomplete type annotations.',
+        inverse=True),
+    'allow_untyped_calls': FlagInfo(
+        arg='--disallow-untyped-calls',
+        doc='Allow calling functions without type annotations from '
+            'typed functions.',
+        inverse=True),
+    'check_untyped_function_bodies': FlagInfo(
+        arg='--check-untyped-defs',
+        doc='Do not check the interior of functions without type annotations.',
+        inverse=False),
+    'strict_optional': FlagInfo(
+        arg='--strict-optional',
+        doc='Enable experimental strict checks related to Optional types. See '
+            '<http://mypy-lang.blogspot.com.es/2016/07/mypy-043-released.html>'
+            ' for an explanation.',
+        inverse=False),
+}
+
+
+def add_param_docs(param_map):
+    """Append documentation from FLAG_MAP to a function's docstring.
```

Comment:
can you add a newline after the `"""`? This is inconsistent with the rest. And while it's not causing issues here it might cause the dedent to fail if it is done in other places by someone who isn't aware of it.